docs/04. Guards.md

```diff
@@ -466,7 +466,7 @@ class IpGuardFactory implements FactoryInterface, MutableCreationOptionsInterfac
 }
 ```
 
-The `MutableCreationOptionsInterface` was introduced in Zend Framework 2.2, and allows your factories to accept
+The `MutableCreationOptionsInterface` was introduced in Laminas Framework 2.2, and allows your factories to accept
```

Comment:
Maybe "Laminas version 2.2" makes more sense than "Laminas Framework 2.2" -- not sure how to deal with the name change retroactively, but as it is, it definitely doesn't feel quite right.